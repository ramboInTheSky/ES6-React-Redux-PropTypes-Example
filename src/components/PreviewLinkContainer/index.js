import styled from 'styled-components';

export default styled.div`
  margin-bottom: 30px;
`;
