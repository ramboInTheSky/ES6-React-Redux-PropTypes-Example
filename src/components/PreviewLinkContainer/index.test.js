import React from 'react';
import { shallow } from 'enzyme';

import PreviewLinkContainer from './';

describe('<PreviewLinkContainer />', () => {
  it('renders correctly', () => {
    expect(shallow(<PreviewLinkContainer />)).toMatchSnapshot();
  });
});
