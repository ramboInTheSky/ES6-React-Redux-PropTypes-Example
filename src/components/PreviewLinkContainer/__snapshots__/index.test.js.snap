// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreviewLinkContainer /> renders correctly 1`] = `
<div
  className="sc-bdVaJa fAeyvH"
/>
`;
